### PR TITLE
task/1347_add_travis_ci

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #1347

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# Author: frb
+
+cd cygnus-common
+mvn test
+mvn clean compile exec:exec assembly:single
+VERSION=$(cat pom.xml | grep version | sed -n '1p' | sed -ne '/<version>/s#\s*<[^>]*>\s*##gp' | sed 's/ //g')
+mvn install:install-file -Dfile=target/cygnus-common-$VERSION-jar-with-dependencies.jar -DgroupId=com.telefonica.iot -DartifactId=cygnus-common -Dversion=$VERSION -Dpackaging=jar -DgeneratePom=true
+cd ../cygnus-ngsi
+mvn test
```
Comment (fgalan) [inline test.sh]: `mvn test` twice?

Comment (frbattid) [inline test.sh]: Yes, one for cygnus-common and other one for cygnus-ngsi (each subproject has its own set of tests).

Comment (fgalan) [inline test.sh]: That explanation should be included as a comment in the file, to avoid other have the same confussion ;)

Comment (frbattid) [inline test.sh]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1349/commits/e59e15731988fd3ac4a8611e86c62b32a44c1232

**Comment by fgalan** [comment]
LGTM

**Pull request merged**